docs/tutorials/data_manipulation/dataframesdataset.md

```diff
@@ -0,0 +1,375 @@
+# Pandas dataframe based dataset
+
+## Introduction
+
+This Tutorial covers how to use gluonts's pandas DataFrame based dataset
+`DataFramesDataset`.
+We create dummy time series data to illustrate how single and
+multiple time series, given in pandas.DataFrame, can be converted
+to `gluonts.dataset.dataframe.DataFramesDataset` and used together with
+gluonts estimators.
+
+
+The minimal requirement to start modelling using gluon-ts's dataframe dataset
```

Comment:
Minor, but: I think we should find a common way (common to the rest of the documentation as well) of referring to the package, whether "GluonTS", "gluonts", or "gluon-ts".

My preference is for "GluonTS" when mentioned in the text.

Reply:
Agree, I think this repo should also be named `gluonts` or `GluonTS` instead of `gluon-ts`. Then everything has one canonical name, maybe sans capitalization.